cmd/git-sync/main.go

```diff
@@ -82,8 +82,10 @@ var flCookieFile = flag.Bool("cookie-file", envBool("GIT_COOKIE_FILE", false),
 var flGitCmd = flag.String("git", envString("GIT_SYNC_GIT", "git"),
 	"the git command to run (subject to PATH search)")
 
-var flHTTPBind = flag.String("http-bind", envString("GIT_SYNC_HTTP_BIND", ""),
-	"the bind address for git-sync's HTTP endpoint")
+var flHTTPBind = flag.String("http-bind", envString("GIT_SYNC_HTTP_BIND", ":8080"),
```

Comment:
I don't think we can make it default-on.  People doing a minor update would get a major behavioral change.

Reply:
Depends on the decision on a global http flag or not. If we have the global gate, then default enable seems reasonable. If not, then I agree with default off.

Reply:
Agreed, this was default-on only after I added the global flag.